Jenkinsfile

```diff
@@ -56,8 +56,46 @@ void unittests(String version) {
                 }
             }
         }
+        stage("pylint{version}") {
```

Comment:
Why do we do this as part of the unittests function. If you do this in a separate function, it can run in parallel to the unittests.

Sharing the docker socket should not be needed for linting?

Reply:
I have complete your suggestion in pull request 348 and cancelling this PR